src/main/scala/org/spurint/maven/plugins/mima/MiMaMojo.scala

```diff
@@ -78,6 +78,9 @@ class MiMaMojo extends AbstractMojo {
   @Parameter(property = "mima.skip", defaultValue = "false")
   private var skip: Boolean = false
 
+  @Parameter(property = "readTimeout", defaultValue = "4000")
+  private var readTimeOut: Int = 4000
```

Comment:
Please make the casing of the variable the same as the property name: `readTimeOut` -> `readTimeout`.

Reply:
Here it is done ;)